source/includes/documentation/_vue_js_elements.md

```diff
@@ -29,30 +29,68 @@ and [JDI vuetify page tests for alerts](https://github.com/jdi-testing/jdi-light
 
 [Vuetify documentation page](https://vuetifyjs.com/en/components/avatars/)
 
+Avatar is located in the following class:
 - __Java__: _com.epam.jdi.light.vuetify.elements.common.Avatar.java_
 
 ```java
-    @Test
-    public void avatarsWithSizeTests() {
-      avatarsWithSize.forEach(avatar -> avatar.is().displayed());
-      avatarsWithSize.get(1).has().text("36");
-      avatarsWithSize.get(2).has().text("48");
-      avatarsWithSize.get(3).has().text("62");
-      avatarsWithSize.get(1).has().size(36);
-      avatarsWithSize.get(2).has().size(48);
-      avatarsWithSize.get(3).has().size(62);
-    }
+@Test(dataProvider = "sizeAvatarsTestData", dataProviderClass = AvatarsTestsDataProvider.class)
+public void sizeAvatarTest(int index, int size) {
+  sizeAvatars.get(index).show();
+  sizeAvatars.get(index).has().size(size)
+        .and().has().height(size)
+        .and().has().heightGreaterThan(size - 10)
+        .and().has().heightLessThan(size + 10)
+        .and().has().width(size)
+        .and().has().widthGreaterThan(size - 10)
+        .and().has().widthLessThan(size + 10);
+  }
 ```
 
 ![Avatars example](../../images/vuetify/avatars.png)
 
-__Avatars__ - graphical representations of users.
+__Vuetify v2.6.14__ code example:
+```html
+<template>
+  <v-row justify="space-around">
+    <v-avatar
+      color="indigo"
+      size="36"
+    >
+      <span class="white--text text-h5">36</span>
+    </v-avatar>
+
+    <v-avatar
+      color="teal"
+      size="48"
+    >
+      <span class="white--text text-h5">48</span>
+    </v-avatar>
+  </v-row>
+</template>
+```
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 
![image](https://user-images.githubusercontent.com/55189259/212974873-ed712e56-6076-466b-9e87-b0f3ff94a693.png)